demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Layouts/layout-sample2_packs_new.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
you can remove this file as `layout` file is not supported for both `packs` and `new`